FireRender.Maya.Src/Lights/PhysicalLight/PhysicalLightData.h

```diff
@@ -84,6 +86,14 @@ struct PhysicalLightData
 	float spotInnerAngle;
 	float spotOuterFallOff;
 
+	// Disk
+	float diskRadius;
+	float diskAngle;
+
+	// Sphere
+
+	float sphereRadius;
+
```

Comment:
I wonder, do we need these new values to be shown in Light Editor?
If yes, light editor template for Physical Light needs to be updated

Reply:
its updated
have a look at AERPRPhysicalLightTemplate.mel  changes

Reply:
need to update LERPRPhysicalLightTemplate.xml as well